src/components/resetpassword-form.js

```diff
@@ -0,0 +1,90 @@
+
+import React from "react";
+import { Input } from "./shared/input";
+
+export class ResetPassword extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentPassword: "",
+      password: "",
+      passwordConfirm: "",
+      touched: {
+        currentPassword: false,
+        password: false,
+        passwordConfirm: false,
+      }
+    };
+  }
+
+  onChangeInputText = e => {
+    const { target } = e;
+    this.setState({
+      [target.id]: [target.value],
+    });
+  }
+
+  handleBlur = (field) => (evt) => {
+    this.setState({
+      touched: { ...this.state.touched, [field]: true },
+    });
+  }
+
+  isValidPassword = () => {
+    const { password, passwordConfirm } = this.state;
+    return !(password === "" || password !== passwordConfirm);
+  };
+
+  isValidCurrentPassword = () =>
+    this.state.currentPassword == window.$password
+
+  isValid = () =>
+    this.isValidPassword() && this.isValidCurrentPassword()
+
+  savePassword = () => {
+    window.$password = this.state.password
```

Comment:
this is not good man. You never must save the password

Reply:
This is only for the practice, a quick way to save the password globally.